Solutions/CiscoDuoSecurity/Analytic Rules/CiscoDuoMultipleUserLoginFailures.yaml

```diff
@@ -0,0 +1,36 @@
+id: 034f62b6-df51-49f3-831f-1e4cfd3c40d2
+name: Cisco Duo - Multiple user login failures
+description: |
+  'Detects when multiple user login failures occurs.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: CiscoDuoSecurity
+    dataTypes:
+      - CiscoDuo
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1078
+query: |
+  let threshold = 10;
+  CiscoDuo
+  | where EventType =~ 'authentication'
+  | where EventResult in~ ('denied', 'failure')
+  | summarize count() by DstUserName, bin(TimeGenerated, 10m)
+  | where count_ > threshold
+  | extend IPCustomEntity = SrcIpAddr, AccountCustomEntity = DstUserName

```

Comment:
Might be a good idea to run this query and check. After the summarize statement it seems like the result will have only three columns DstUserName, TimeGenerated and count_. You cannot extend IPCustomEntity and use it in Entity mapping if the column is not there.

Reply:
fixed